src/test/java/uk/gov/companieshouse/api/testdata/service/impl/CompanyProfileServiceImplTest.java

```diff
@@ -178,14 +192,14 @@ void delete() {
         assertTrue(this.companyProfileService.delete(COMPANY_NUMBER));
         verify(repository).delete(companyProfile);
     }
-    
+
```

Comment:
In future, try to keep general reformatting/WS changes out of code-change PRs, perhaps confining them to a reformat PR.